### PR TITLE
feat: Add apps/scraped_imports

### PR DESCRIPTION
**Pull request by jbrundage**
@GordonSmith this "apps" entry is a handy node script that makes it easy to browse widget imports

### CODE REVIEW

**Comment by jbrundage** [comment]
image

File: apps/scraped_imports/app.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const packageArr = fs.readdirSync(packagesPath);
+
+const _imports = {};
+
+packageArr.forEach(package=>{
+    try{
+        const srcPath = `${packagesPath}\\${package}\\src\\`;
+        const widgetArr = fs.readdirSync(srcPath);
+        widgetArr.forEach(widget=>{
+            const widgetClass = `${package}_${widget}`;
```
Comment (GordonSmith) [inline apps/scraped_imports/app.js]: Needs to be recursive as some `src` folders have nested folders (like in map)

Comment (GordonSmith) [reply apps/scraped_imports/app.js]: IOW needs to test if "widget" is a file or folder

File: apps/scraped_imports/scraped_imports.json [hunk 2]
```diff
@@ -0,0 +1,1300 @@
...
+        "import { arc as d3Arc } from \"d3-shape\";",
+        "import \"../src/RadialBar.css\";"
+    ],
+    "chart_Scatter.ts": [
+        "import { INDChart, ITooltip } from \"@hpcc-js/api\";",
+        "import { InputField } from \"@hpcc-js/common\";",
+        "import { extent as d3Extent } from \"d3-array\";",
+        "import { scaleLinear as d3ScaleLinear, scaleLog as d3ScaleLog, scalePow as d3ScalePow, scaleSqrt as d3ScaleSqrt } from \"d3-scale\";",
+        "import { select as d3Select } from \"d3-selection\";",
+        "import {",
```
Comment (GordonSmith) [inline apps/scraped_imports/scraped_imports.json]: Issue?

File: apps/scraped_imports/scraped_imports.json [hunk 3]
```diff
@@ -0,0 +1,1300 @@
...
+    ],
+    "codemirror_DOTEditor.ts": [
+        "import { Editor } from \"./Editor\";"
+    ],
+    "codemirror_ECLEditor.css": [
+        "important;color:white!",
+        "important;}",
+        "important;}",
+        "important; }"
+    ],
```
Comment (GordonSmith) [inline apps/scraped_imports/scraped_imports.json]: Issue

File: apps/scraped_imports/app.js (hunk 1 above)
Comment (GordonSmith) [reply apps/scraped_imports/app.js]: Also needs to check the file type (css v ts etc,)

**Comment by GordonSmith** [comment]
@jbrundage bump

File: apps/scraped_imports/app.js (hunk 1 above)
Comment (GordonSmith) [reply apps/scraped_imports/app.js]: @jbrundage bump

File: apps/scraped_imports/scraped_imports.json (hunk 2 above)
Comment (GordonSmith) [reply apps/scraped_imports/scraped_imports.json]: @jbrundage bump

File: apps/scraped_imports/scraped_imports.json (hunk 3 above)
Comment (GordonSmith) [reply apps/scraped_imports/scraped_imports.json]: @jbrundage bump

File: apps/scraped_imports/scraped_imports.json (hunk 2 above)
Comment (jbrundage) [reply apps/scraped_imports/scraped_imports.json]: fixed

File: apps/scraped_imports/scraped_imports.json (hunk 3 above)
Comment (jbrundage) [reply apps/scraped_imports/scraped_imports.json]: fixed

**Comment by jbrundage** [comment]
@GordonSmith I've revised the code and the output is looking good.

**Pull request merged**